app/_layouts/extension.html

```diff
@@ -283,14 +408,37 @@
   {% if config_required_fields_yaml == 'config: ' %}config:
     <optional_parameter>: <value>{% else %}{{ config_required_fields_yaml }}{% endif %}
 ```
-{% endnavtab %}
-{% endunless %}
-{% endnavtabs %}
-{% endif %}
 
 `<route>` is the `id` or `name` of the Route that this plugin configuration
   will target.
+{% endnavtab %}
+{% endunless %}
+{% unless page.params.konnect_examples == false %}
+{% navtab Konnect %}
+Configure this plugin on a Route:
+
+1. In Konnect, select the Service from the ServiceHub page.
+2. Scroll down to **Versions** and select the version.
+3. Select the route.
```

Comment:
inconsistent capitalization of "route". Shane had mentioned that Kong docs have too much initial caps that are "reader speed bumps". Should we even capitalize Service and Route anymore? Unless it is reflecting a specific UI element that is initial-capped?